ddtelemetry/examples/tm-metrics-worker-test.rs

```diff
@@ -52,7 +52,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     let dist_metric = handle.register_metric_context(
         "test_telemetry.dist".into(),
         Vec::new(),
-        data::metrics::MetricType::Distribution,
+        data::metrics::MetricType::Sketch,
```

Comment:
nitpicking - Isn't it still a distribution?

Reply:
You're right, I renamed it so it's called Distribution for public interfaces, and sketche is just an implementation detail